src/services/functionAppService.ts

```diff
@@ -149,7 +149,13 @@ export class FunctionAppService extends BaseService {
 
     const functionZipFile = this.getFunctionZipFile();
     const uploadFunctionApp = this.uploadZippedArfifactToFunctionApp(functionApp, functionZipFile);
-    const uploadBlobStorage = this.uploadZippedArtifactToBlobStorage(functionZipFile);
+    // If `runFromBlobUrl` is configured, the artifact will have already been uploaded
+    const uploadBlobStorage = (this.deploymentConfig.runFromBlobUrl)
```

Comment:
To support rollback - don't we already ALWAYS upload the artifact to blob storage?

Reply:
Then its really just a configuration regarding whether the `RUN_FROM_PACKAGE` is set to 1 vs the SAS URL ???